Dockerfile

```
@@ -0,0 +1,37 @@
+ENV PRESETS="release"
+ENV DESTDIR=/app
+ENV PREFIX=/
+
+WORKDIR /build
+COPY . .
+
+RUN make install-release
+
+FROM ubuntu:22.04
+
+RUN apt-get update && apt-get install -y \
```

Hadolint (2.14.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)